src/status_im/chat/models.cljs

```diff
@@ -3,8 +3,8 @@
             [taoensso.timbre :as log]
             [status-im.multiaccounts.model :as multiaccounts.model]
             [status-im.chat.models.message-list :as message-list]
-            [status-im.chat.models.delete-message :as delete-message]
-            [status-im.chat.models.delete-message-for-me :as delete-for-me]
+            [status-im2.contexts.chat.messages.message.delete-message.events :as delete-message]
+            [status-im2.contexts.chat.messages.message.delete-message-for-me.events :as delete-for-me]
```

Comment:
what's the point of separating delete-message and delete-message-for-me ?

Reply:
but im fine with that, just wondering

Reply:
will do

Reply:
? i'm ok to keep them separate :)

Reply:
I'll leave this one here and go back to this when adding the deleted by XXX feature